tests/rados/test_omap_entries.py

```diff
@@ -25,13 +27,23 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosOrchestrator(node=cephadm)
     pool_obj = PoolFunctions(node=cephadm)
+    client_node = ceph_cluster.get_nodes(role="client")[0]
 
     omap_target_configs = config["omap_config"]
+    crash_pool_name = config["crash_config"]["pool_name"]
+    if not rados_obj.create_pool(pool_name=crash_pool_name):
+        log.error(f"Failed to create pool-{crash_pool_name}")
+        return 1
+    # put the object
 
-    # Creating pools and starting the test
-    for omap_config_key in omap_target_configs.keys():
-        omap_config = omap_target_configs[omap_config_key]
-        try:
+    cmd_put_obj = f"rados put -p {crash_pool_name} obj1 /etc/hosts"
+    client_node.exec_command(cmd=cmd_put_obj, sudo=True)
+    cmd_clearomap = f"rados clearomap -p {crash_pool_name} obj1"
```

Comment:
The command needs to be updated. Please discuss with Harsh for the workflow.

Reply:
Discuss with Harsh and updated the workflow.